packages/amplication-client/src/Resource/git/GitActions/GitProviderConnection.tsx

```diff
@@ -53,7 +54,7 @@ export default function GitProviderConnection({
   }, [provider]);
 
   const { currentWorkspace } = useContext(AppContext);
-  const upgradeLink = `/${currentWorkspace?.id}/purchase`;
+  const contactUsLink = "https://amplication.com/contact-us";
```

Comment:
take it out of the component, it's static

Reply:
there is no reason to put it in a different contact file because
- this is the only place we use it
- we don't have one in this scope and creating a constant file only for this reason it's unnecessary in my opinion